src/fairseq2/datasets/preference.py

```diff
@@ -63,6 +63,7 @@ def create_reader(
         max_num_batches: int | None = None,
         num_accumulate: int = 1,
         num_prefetch: int = 1,
+        mask_src_tokens: bool = True,
```

Comment:
Nit: in the rest of the code base we use `source` instead of `src` in variable names. So `mask_source_tokens` would be more consistent.

Reply:
Also, can you please a brief docstr for this parameter?

Reply:
thanks for docstr catch!